frontend/src/employee-frontend/components/person-profile/income/IncomeItemBody.tsx

```diff
@@ -43,6 +49,32 @@ const IncomeItemBody = React.memo(function IncomeItemBody({ income }: Props) {
             ? i18n.personProfile.income.details.originApplication
             : income.updatedBy}
         </span>
+        {income.notes === 'created automatically from application' &&
```

Comment:
It would be better have the backend signal this with a flag rather than this string deduction, but ok for a hotfix if it proves to be to big of a change